### PR TITLE
Dockerhub push gh actions

### PR DESCRIPTION
**Pull request by jaredb96**

### CODE REVIEW

File: .github/workflows/dockerhub-push.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: HMDA Docker Hub Image Push
```
Comment (tptignor) [inline .github/workflows/dockerhub-push.yml:1]: General question: what fills in all the template values?

Comment (jaredb96) [reply .github/workflows/dockerhub-push.yml:1]: Can you clarify this question

File: .github/workflows/dockerhub-push.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build image of HMDA Platform only
+        run: |
+            sbt "project hmda-platform" dockerPublishLocalSkipTests
```
[the remark this answers is not in the document]
Comment (jaredb96) [reply .github/workflows/dockerhub-push.yml]: Edited this line to run tests

File: .github/workflows/dockerhub-push.yml (hunk 1 above)
Comment (tptignor) [reply .github/workflows/dockerhub-push.yml:1]: Where do all the values in curly brackets come from?

**Review by tptignor — APPROVED** [review]
LGTM

**Pull request merged**